### PR TITLE
Improve kafka schema registry authorization

### PR DESCRIPTION
**Pull request by juha-aiven**
# What changed, and why it matters

The other commit fixes a typo.

The other commits adds Terraform usage info.

### CODE REVIEW

File: docs/products/kafka/howto/manage-schema-registry-authorization.rst [hunk 1]
```diff
@@ -23,3 +23,13 @@ Here's an example of how to add an ACL entry granting a user named ``user_1`` re
 .. Tip::
     
     The Aiven CLI command ``service schema-registry-acl-list`` allows you to list the ACL entries already defined. ``service schema-registry-acl-delete`` allows you to delete an ACL entry.
+
+Additionally :doc:`Aiven Aiven Terraform provider </docs/tools/terraform>` supports managing Karapace schema registry authorization ACL entries by using the resource `aiven_kafka_schema_registry_acl <https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/kafka_schema_registry_acl>`_::
+
+   resource "aiven_kafka_schema_registry_acl" "my_resource" {
+     project      = "PROJECT_NAME"
```
Comment (mhoffm-aiven) [inline docs/products/kafka/howto/manage-schema-registry-authorization.rst:30]: consider references

**Comment by mhoffm-aiven** [comment]
Please consider using `terraform` lexer for the terraform code block too. Otherwise looks good to me.

File: docs/products/kafka/howto/manage-schema-registry-authorization.rst [hunk 2]
```diff
@@ -23,3 +23,15 @@ Here's an example of how to add an ACL entry granting a user named ``user_1`` re
 .. Tip::
     
     The Aiven CLI command ``service schema-registry-acl-list`` allows you to list the ACL entries already defined. ``service schema-registry-acl-delete`` allows you to delete an ACL entry.
+
+Additionally :doc:`Aiven Aiven Terraform provider </docs/tools/terraform>` supports managing Karapace schema registry authorization ACL entries by using the resource `aiven_kafka_schema_registry_acl <https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/kafka_schema_registry_acl>`_
```
Comment (dewan-ahmed) [inline docs/products/kafka/howto/manage-schema-registry-authorization.rst:27]: Aiven Aiven Terraform provider --> repeated words.

Suggestion: Aiven Terraform Provider

**Review by dewan-ahmed — COMMENTED** [review]
I'm discussing with our Vale expert to understand how to avoid the Vale warnings from this PR. This happens on line27 where aiven, kafka, acl are written within a link. I'll come back to this PR once I hear more on this.

File: docs/products/kafka/howto/manage-schema-registry-authorization.rst (hunk 2 above)
Comment (dewan-ahmed) [reply docs/products/kafka/howto/manage-schema-registry-authorization.rst:27]: @juha-aiven FYI ^^

**Review by TibsAtWork — APPROVED** [review]
Linter is now happy, so I'm going to approve/merge

**Pull request merged**